docs/components.md

```diff
@@ -245,6 +260,8 @@ export default class extends TemplateComponent {
 
 We do not need `valueElement` property and `update` method. If property used in `template` method is reactive, DOM will be updated automatically.
 
-Sometimes, you would like to do something after whole template would be rerendered. Then you can use `this.$requestUpdate()` method that returns `Promise` that will resolve after rerender.
+Sometimes, you would like to force template to rerender, because of some none-reactive change. Then you can use `this.$requestUpdate()` method that returns `Promise` that will resolve after rerender.
```

Comment:
```suggestion
Sometimes, you would like to force template to rerender, because of some non-reactive change. Then you can use `this.$requestUpdate()` method that returns `Promise` that will resolve after rerender.
```